src/traces/scattergl/convert.js

```diff
@@ -190,7 +190,7 @@ function convertTextStyle(gd, trace) {
     if(
         isArrayOrTypedArray(tfs) ||
         Array.isArray(tff) ||
-        Array.isArray(tfw) ||
+        Lib.isArrayOrTypedArray(tfw) ||
```

Comment:
is this different from `isArrayOrTypedArray` above?

Reply:
They are identical. Good eye. Addressed in [82863fd](https://github.com/plotly/plotly.js/pull/6990/commits/82863fdf0cb4ccfdd610567670b34bc3751380c9).